fission-web-server/env.yaml.example

```diff
@@ -22,7 +22,7 @@ storage:
     database: web_api
 
 auth:
-  fission_did: did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4
+  fission_did: did:key:z2DSW536bcWxPGuz7ZMnXdju64pBoWrybTyzTqWYWa7EjsB
```

Comment:
Same key but in updated format

Reply:
I'll have to update this in my pizza instance, right?